### ISSUE TITLE
Example for development options

### ISSUE THREAD

**Issue by pcambra** [rwjblue/ember-cli-content-security-policy#76]
Maybe it would be useful to include in the README a open to all development example, something like:

```
if (environment === 'development') {
    ENV.contentSecurityPolicy = {
      'default-src': ["*"],
      'script-src':  null,
      'font-src':    null,
      'connect-src': null,
      'img-src':     null,
      'style-src':   null,
      'media-src':   null
    };
}
```

**Comment by sandstrom** [comment]
@pcambra Can you PR something for the *Examples* section of the readme?

**Comment by jelhan** [comment]
I would recommend disabling or uninstalling it all. I don't see the use case for keeping it enabled but allowing all requests. Maybe this one was just motivated by the confusing configuration options? Could this be closed as there is an `enabled` option (#94) now?

**Comment by rwjblue** [comment]
Ya, agreed @jelhan
